client/src/components/editor/RichTextEditor.css

```
@@ -1,21 +1,55 @@
+:root {
+  --banner-height: 50px
+}
 .editor-container {
   border: 1px solid #ccc;
   border-radius: 4px;
 }
 
+/* Style used to make rich-text-editor to full-size */
+.editor-container-fullsize {
+  position: absolute; 
+  /* In full size mode the richtext-editor start position, just below the topbar */
+  top: var(--banner-height);
+  left: 0;
+  max-width: none;
+  max-height: calc(100% - var(--banner-height));
+  width: 100%;
+  height: 100%;
+  background: whitesmoke;
+  z-index: 1201;
+  /* Hide scrollbars in both x and y directions */
+  overflow-y: hidden;
+  overflow-x: hidden;
```

```suggestion
  overflow: hidden;
```
If you write like this, it will be much better

---

You can use overflow: hidden istead of overflow-x and overflow-y